tests/cla_chain_test.sh

```diff
@@ -24,6 +24,7 @@ PORT_NODE1=3000
 PORT_NODE2=3001
 PORT_NODE3=3002
 PORT_NODE4=3003
+PORT_NODE5=3004
```

Comment:
since this is test script is mainly to test all the different CLAs in general, it might be better to have an extra script for tcpcl itself where different extension can be tested? makes debugging and maintaining it easier in the long run